js/huobi.js

```diff
@@ -5140,7 +5173,10 @@ module.exports = class huobi extends Exchange {
         await this.loadMarkets ();
         const marginMode = this.safeString2 (this.options, 'defaultMarginMode', 'marginMode', 'isolated');
         const defaultSubType = this.safeString (this.options, 'defaultSubType', 'inverse');
-        const [ marketType, query ] = this.handleMarketTypeAndParams ('fetchPositions', undefined, params);
+        let [ marketType, query ] = this.handleMarketTypeAndParams ('fetchPositions', undefined, params);
+        if (marketType === 'spot') {
```

Comment:
I added this because it was failing the test for fetchPositions, as this should only be called for contracts, but it was defaulting to a `spot` market type.